src/middlewares/auth-middleware.ts

```diff
@@ -48,104 +47,24 @@ async function authentication(
     }
     span.setDisclosedAttribute("user cookie found", true);
 
-    const { access_token: currentAccessToken, sub } = userCookie;
+    const { access_token: currentAccessToken } = userCookie;
 
     await decryptVerifyAccessToken(currentAccessToken).catch(async (error) => {
       if (error.name === "TokenExpiredError") {
         span.setDisclosedAttribute("is access_token expired", true);
 
-        const user = await getDBUserFromSub(sub).catch((error) => {
-          span.setDisclosedAttribute("database reachable", false);
-          throw webErrorFactory({
-            ...webErrors.databaseUnreachable,
-            parentError: error,
-          });
-        });
-        span.setDisclosedAttribute("database reachable", true);
-
-        if (!user) {
-          span.setDisclosedAttribute("user found on DB", false);
-          throw new NoDBUserFoundError();
+        if (request.url && request.url.includes("/api/")) {
```

Comment:
```suggestion
        if (request.url && request.url.startsWith("/api/")) {
```

Seems safer to me, what do you think?

Reply:
I feel like a fool, I didn't see your review until this afternoon. (Once again, I didn't receive notifications on Slack for the comments... 😖 )
But regarding the comment, I agree, it will be safer I think 🙂